modules/AWS/rds/locals.tf

```diff
@@ -1,5 +1,7 @@
 locals {
-  product_db_name    = var.product_db_name == null ? var.product : var.product_db_name
+  product_db_name   = var.product
+  rds_instance_name = var.snapshot_identifier == null ? var.product : null
```

Comment:
`rds_instance_name` will make me think it's `module.rds.identifier`. I would suggest naming the variable `db_name`.

Reply:
I renamed `rds_instance_id` to `rds_instance_indentifier` in bitbucket snapshot PR to make the variable name more clear. Considering that change this name will not be confused anymore.